Project.toml

```diff
@@ -57,22 +63,22 @@ LegendHDF5IO = "0.1"
 LegendTextIO = "0.1"
 LsqFit = "0.13, 0.14, 0.15"
 MJDSigGen = "0.2"
-Parameters = "0.12"
+Parameters = "0.12, 0.13"
 Polynomials = "2, 3, 4"
 PropDicts = "0.2"
 RadiationDetectorDSP = "0.2.1"
 RadiationDetectorSignals = "0.3"
 RadiationSpectra = "0.5"
-Random123 = "1.2"
+Random123 = "1.2, 1.7" # Compatible with 1.7?
```

Comment:
The syntax "1.2" means "everything 1.x.x, starting 1.2.0". You don't need to explicitly add `1.7` (in fact, I would remove this here)